libs/data-access/src/lib/tp/job-listings/tp-job-listing.fragment.graphql

```diff
@@ -22,4 +22,8 @@ fragment AllTpJobListingFields on TpJobListing {
   contactEmailAddress
   contactPhoneNumber
   isFromCareerPartner
+  contactFirstName
+  contactLastName
+  contactEmailAddress
+  contactPhoneNumber
```

Comment:
These are duplicated. I believe we're addressing this in another PR or even addressed already. Rebasing your branch could fix the issue.

Reply:
@helloanil, I pushed a commit to fix it. There should be no duplicated contact details.